<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>637</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QTableView" name="historyTableView">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="bottomHalfLayout">
       <item>
        <layout class="QVBoxLayout" name="currentTrackAndProgressLayout">
         <item>
          <widget class="QLabel" name="artistTitleLabel">
           <property name="styleSheet">
            <string notr="true">font-size: 11pt;</string>
           </property>
           <property name="text">
            <string>Artist - Title</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="PMP::TrackProgressWidget" name="trackProgress" native="true">
           <property name="minimumSize">
            <size>
             <width>8</width>
             <height>8</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="currentPositionAndPersonalModeLayout">
         <item>
          <widget class="QPushButton" name="trackInfoButton">
           <property name="text">
            <string>Track info</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="lengthLabel">
             <property name="text">
              <string>Length:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="lengthValueLabel">
             <property name="text">
              <string>unknown</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="positionLabel">
             <property name="text">
              <string>Position:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="positionValueLabel">
             <property name="text">
              <string>unknown</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="playingModeLabel">
             <property name="text">
              <string>unknown mode</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="userPlayingForLabel">
             <property name="text">
              <string>Timmy</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="toPersonalModeButton">
           <property name="text">
            <string>Kevin</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="toPublicModeButton">
           <property name="text">
            <string>Public</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="playbackAndVolumeControlsLayout">
         <item>
          <widget class="QPushButton" name="playButton">
           <property name="icon">
            <iconset resource="mainwindow.qrc">
             <normaloff>:/mediabuttons/Play.png</normaloff>:/mediabuttons/Play.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pauseButton">
           <property name="icon">
            <iconset resource="mainwindow.qrc">
             <normaloff>:/mediabuttons/Pause.png</normaloff>:/mediabuttons/Pause.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="skipButton">
           <property name="icon">
            <iconset resource="mainwindow.qrc">
             <normaloff>:/mediabuttons/Skip.png</normaloff>:/mediabuttons/Skip.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="playStateLabel">
           <property name="text">
            <string>playing/paused</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="volumeLabel">
           <property name="text">
            <string>Volume:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="volumeValueLabel">
           <property name="text">
            <string>100</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="volumeDecreaseButton">
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>-5</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="volumeIncreaseButton">
           <property name="text">
            <string>+5</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="queueTopControlsLayout">
         <item>
          <widget class="QFrame" name="horizontalFrame">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QPushButton" name="insertBreakButton">
              <property name="text">
               <string>Insert break</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="queueLengthLabel">
           <property name="text">
            <string>Queue length:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="queueLengthValueLabel">
           <property name="text">
            <string>unknown</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="queueTableView">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="dragDropOverwriteMode">
          <bool>false</bool>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="dynamicModeControlsLayout">
         <item>
          <widget class="QCheckBox" name="dynamicModeCheckBox">
           <property name="text">
            <string>dynamic mode</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="queueOperationsLabel">
           <property name="text">
            <string>queue operations:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="expandButton">
           <property name="text">
            <string>Expand</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="trimButton">
           <property name="text">
            <string>Trim</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="dynamicModeLine2Layout">
         <item>
          <widget class="QLabel" name="dynamicModeTrackSelectionLabel">
           <property name="text">
            <string>dynamic mode track selection:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>avoid repeating the same track within:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="noRepetitionComboBox">
           <property name="maxVisibleItems">
            <number>15</number>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToContents</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="dynamicModeLine3Layout">
         <item>
          <widget class="QLabel" name="highScoredTracksWaveLabel">
           <property name="text">
            <string>high-scored tracks wave:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="startWaveButton">
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="waveProgressValueLabel">
           <property name="text">
            <string>0 / 20</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="terminateWaveButton">
           <property name="text">
            <string>Terminate</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PMP::TrackProgressWidget</class>
   <extends>QWidget</extends>
   <header>gui-remote/trackprogresswidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
